bindings/go/examples/gas_sponsor/main.go

```diff
@@ -30,7 +54,9 @@ func main() {
 		nil,
 	)
 
-	builder.Sponsor(sponsor)
+	gasObjId := objIdFromHex("0x0b0270ee9d27da0db09651e5f7338dfa32c7ee6441ccefa1f6e305735bcfc7ab")
+	gasObjIds := []*sdk.ObjectId{gasObjId}
+	builder.Gas(gasObjIds).Sponsor(sponsor)
```

Comment:
???

Reply:
Aligned the go example with the other ones.

Reply:
Ahhhh sorry I literally just was looking at a merge commit and got confused, which is why I resolved this immediately 🙈